packages/contracts-bedrock/scripts/deploy/Deploy.s.sol

```diff
@@ -435,34 +450,31 @@ contract Deploy is Deployer {
         addr_ = address(manager);
     }
 
-    /// @notice Deploy the ProxyAdmin
-    function deployProxyAdmin(bool _isSuperchain) public broadcast returns (address addr_) {
-        string memory proxyAdminName = _isSuperchain ? "SuperchainProxyAdmin" : "ProxyAdmin";
-
-        console.log("Deploying %s", proxyAdminName);
-
-        // Include the proxyAdminName in the salt to prevent a create2 collision when both the Superchain and an OP
-        // Chain are being setup.
+    /// @notice Deploys the ProxyAdmin contract. Should NOT be used for the Superchain.
```

Comment:
Why is this, i.e. why can we no longer reuse deployProxyAdmin?

Reply:
It's clunky with the AddressManager conditional to reuse it, but wondering if there was another reason

Reply:
Both [`DeploySuperchain`](https://github.com/ethereum-optimism/optimism/blob/sc/fix-kontrol-deploy/packages/contracts-bedrock/scripts/DeploySuperchain.s.sol#L323-L328) and [`OPCM`](https://github.com/ethereum-optimism/optimism/blob/sc/fix-kontrol-deploy/packages/contracts-bedrock/src/L1/OPContractsManager.sol#L243-L244) deploy a distinct `ProxyAdmin`.

Reply:
> why can we no longer reuse deployProxyAdmin?

We want to remove that and replace it with the OPCM functionality.

Reply:
Yea, I was trying to understand is that if reusing the `deployProxyAdmin` method (not reusing the ProxyAdmin) was related to the bug, or if that change is just cleanup towards that goal. Rereading the list of bugs from discord:

> SuperchainProxyAdmin is deployed twice. Once as part of deployProxyAdmin and once as part of DeploySuperchain. Would be caught at runtime except save("superchainProxyAdmin", address(dso.superchainProxyAdmin())); uses lowercase "superchainProxyAdmin" instead of "SuperchainProxyAdmin" which bypasses the check that the same contract is deployed twice.

The answer is that it was a bug, so resolving